### PR TITLE
Jetpack SSO: Update SSO component to use blog details

### PR DESCRIPTION
**Pull request by ebinnion**
Adds site specific details to the `/jetpack/sso` route.

### CODE REVIEW

File: client/state/jetpack-connect/test/reducer.js [hunk 1]
```diff
@@ -29,6 +29,28 @@ import reducer, {
...
+			ico: '',
+		},
+		URL: 'https://website.com',
+		is_vip: false,
+		admin_url: 'https://website.com/wp-admin'
+	}
+};
+
+const falseSSOValidation = {
+	type: JETPACK_CONNECT_SSO_VALIDATION_SUCCESS,
```
Comment (lezama) [inline client/state/jetpack-connect/test/reducer.js]: Shouldn't this action be named `JETPACK_CONNECT_SSO_VALIDATION_FAILED` or something like that?

Comment (ebinnion) [inline client/state/jetpack-connect/test/reducer.js]: In this case, the request was successful and is returning a boolean value. So, I'm not sure that `JETPACK_CONNECT_SSO_VALIDATION_FAILED` fits either.

Comment (lezama) [inline client/state/jetpack-connect/test/reducer.js]: :+1:

**Comment by lezama** [comment]
:shipit:

**Pull request merged**